src/themes/OLH/templates/elements/journal/issue_list_by_decade.html

```diff
@@ -0,0 +1,24 @@
+<div class="large-12 columns">
+    <p>
+        {% blocktrans %}
+            Issues are grouped by decade. Select a decade to view the issues
+            published in during that decade.
```

Comment:
There's a typo with "in during". Same comment about the templates in clean and material.